config/locales/client.en.yml

```diff
@@ -597,7 +597,7 @@ en:
       topics_entered: "topics entered"
       post_count: "# posts"
       confirm_delete_other_accounts: "Are you sure you want to delete these accounts?"
-      powered_by: "powered by <a href='https://ipinfo.io'>ipinfo.io</a>"
+      powered_by: "powered by <a href='https://maxmind.com'>maxmind.com</a>"
```

Comment:
I don't think we have any attribution requirements here, we can remove this imo. We do not want people to think we are uploading IPs to maxmind

Reply:
According to License section of [this document](https://dev.maxmind.com/geoip/legacy/geolite/), I believe we do.

Reply:
That only says we need to say so in ads and documentations ;)